cmd/app.go

```diff
@@ -1308,3 +1299,24 @@ func checkShowSubquestionIf(q managementClient.Question, answers map[string]stri
 	}
 	return false
 }
+
+// return answers and empty values or values and empty answers to match UI flow
```

Comment:
Couldn't both be returned here ? Or would both not be passed in

Reply:
I will make the comment more clear, but essentially if you pass in answers OR values, the other one should be empty, if you pass in BOTH in the cli command - both should return values

Reply:
Thats more clear, although code looks like if you passed both it would return both, thats why comment confused me.